README.md

```diff
@@ -30,10 +30,10 @@ For local development, you can spin up a local instance of Keycloak, backed by P
 Once the environment is ready, you can run the acceptance tests after setting the required environment variables:
 
 ```
-KEYCLOAK_CLIENT_ID=terraform \
-KEYCLOAK_CLIENT_SECRET=884e0f95-0f42-4a63-9b1f-94274655669e \
-KEYCLOAK_REALM=master \
-KEYCLOAK_URL="http://localhost:8080" \
+export KEYCLOAK_CLIENT_ID=terraform \
+export KEYCLOAK_CLIENT_SECRET=884e0f95-0f42-4a63-9b1f-94274655669e \
+export KEYCLOAK_REALM=master \
+export KEYCLOAK_URL="http://localhost:8080" \
```

Comment:
This works as-is without the `export` statements.  Is there a reason this needed to be changed?

Reply:
@alexashley made me do it.